pages/common/subl.md

```diff
@@ -0,0 +1,19 @@
+# subl
+
+> Sublime Text, a sophisticated text editor for code, markup and prose.
```

Comment:
Teensy grammatical correction: `markup and prose` => `markup, and prose`. ([ref](https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma))